Source/WebKit/UIProcess/Cocoa/PageClientImplCocoa.mm

```diff
@@ -58,22 +58,23 @@
 
 void PageClientImplCocoa::obscuredContentInsetsDidChange()
 {
-    [m_webView _recalculateViewportSizesWithMinimumViewportInset:[m_webView minimumViewportInset] maximumViewportInset:[m_webView maximumViewportInset] throwOnInvalidInput:NO];
+    RetainPtr webView = m_webView.get();
+    [webView _recalculateViewportSizesWithMinimumViewportInset:[webView minimumViewportInset] maximumViewportInset:[webView maximumViewportInset] throwOnInvalidInput:NO];
 }
 
 void PageClientImplCocoa::themeColorWillChange()
 {
-    [m_webView willChangeValueForKey:@"themeColor"];
+    [webView() willChangeValueForKey:@"themeColor"];
```

Comment:
Maybe we should rename this to `protectedWebView` in a follow up?

Reply:
We can but it is similar as in my other PR, there would only be a `protectedWebView()` getter, no `webView()` getter then, since `WeakObjCPtr<T>::get()` returns a `RetainPtr`.

Reply:
Hm... I see. I guess because objective-c weakptr is thread safe?